soot-infoflow-android/src/soot/jimple/infoflow/android/source/AndroidSourceSinkManager.java

```diff
@@ -916,15 +942,25 @@ public void initialize() {
 				SourceSinkDefinition sourceSinkDef = entry.getValue();
 				if (sourceSinkDef instanceof MethodSourceSinkDefinition) {
 					SootMethod sm = Scene.v().grabMethod(entry.getKey());
+
 					if (sm != null)
 						sourceMethods.put(sm, sourceSinkDef);
+					else{ //in case method signature doesn't contain return type
```

Comment:
There should rather be a flag whether this is a method with a return type or not. The parser can set such a flag, because it knows whether the format supports a return type or not.

Reply:
Done